src/Xamarin.Android.Build.Tasks/Utilities/UnhandledExceptionLogger.cs

```diff
@@ -67,8 +67,6 @@ public static void LogUnhandledException (this TaskLoggingHelper log, string pre
 				log.LogCodedError (prefix + "7027", ex.ToString ());
 			else if (ex is FileNotFoundException)		// IOException
 				log.LogCodedError (prefix + "7028", ex.ToString ());
-			else if (ex is PipeException)			// IOException
```

Comment:
I guess this means nobody is hitting it yet?

Reply:
As mentioned above:
 > The XA7029 error code was only added in d16-4, and as a result we don't yet have any hits in the wild. However, since this was only recently released, I believe it should be relatively harmless to remove.

We can circle back once we have some more telemetry data from d16-4, or go with the proposed reflection based change now if that's preferable?

Reply:
Fair enough.  d16-4 only hit stable ~3 days ago...